projects/policekillings/components/widgets/BrutalityOverTime.js

```diff
@@ -0,0 +1,55 @@
+import React, { useMemo } from "react";
+import { VictoryChart, VictoryLine, VictoryAxis } from "victory";
+import { useMeasure } from "react-use";
+import { COLORS } from "styles/constants";
+
+function transformData(data) {
```

Comment:
Similar note to before: how is the data being transformed? Can you think of a more descriptive name about the transformation?

Additionally, if you find every component that leverages these `VictoryChart` utility transform functions, it could be worthwhile to make a separate utility file that the different components could reference.